pandas/core/groupby.py

```diff
@@ -2368,6 +2368,13 @@ def _get_grouper(obj, key=None, axis=0, level=None, sort=True,
     # axis of the object
     if level is not None:
         if not isinstance(group_axis, MultiIndex):
+            if isinstance(level, list) or isinstance(level, range):
+                if len(level) > 1:
```

Comment:
Now that I think about it: should we raise an error when given an empty iterable for `level` instead of making it `level=0`?